src/components/organisms/OurSupportersSection/OurSupportersSection2.js

```diff
@@ -178,122 +42,24 @@ export default function OurSupportersSection({
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter1} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter2} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter3} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter4} ImageStyleProps="" />
-            </div>
-          </div>
-        </div>
 
-        <div>
-          <div className="w-full flex justify-center my-14" >
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter5} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter6} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter7} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter8} ImageStyleProps="" />
-            </div>
-          </div>
-        </div>
-        
-        <div>
-          <div className="w-full flex justify-center my-14" >
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter9} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter10} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter11} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter12} ImageStyleProps="" />
-            </div>
-          </div>
-        </div>
+        {/* Supporters */}
 
-        <div>
-          <div className="w-full flex justify-center my-14" >
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter13} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter14} ImageStyleProps="" />
-            </div>
-            <div className="flex mx-10">
-              <ImageAtomLocal imagesize="" alt="" border="" src={Supporter15} ImageStyleProps="" />
+        <div className="w-full flex justify-center flex-wrap">
+          {Array.from({ length: 27 }, (_, i) => (
+            <div className="flex mx-0 justify-center" key={i} style={{ flexBasis: '25%' }}>
```

Comment:
Have you tried using `basis-1/4` instead of `style={{ flexBasis: '25%' }}`? According to Tailwind docs, it translates to `flex-basis: 25%`.

Reply:
👍🏼